### PR TITLE
updated library code

### PR DESCRIPTION
**Pull request by Jobin1000**

### CODE REVIEW

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```
Comment (samuel-abhishek) [inline .idea/.gitignore]: .idea folder should not be committed as they are local IDE config files. Please delete the .idea folder and update the .gitignore file likewise.

File: .gitignore [hunk 2]
```diff
@@ -1,8 +1,15 @@
+*.iml
 .gradle
 /local.properties
-/.idea/workspace.xml
+/.idea/caches
 /.idea/libraries
-.idea
+/.idea/modules.xml
```
Comment (samuel-abhishek) [inline .gitignore]: Delete all .idea subfolder entries and include the whole .idea folder.

File: pulldownlayout/src/main/java/com/w4lle/library/PullDownView.java [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+            topView.getLayoutConfig().height = topViewHeight;
+            isonce = true;
+            layoutParams = containerView.getLayoutConfig();
+            oldTopMargin = layoutParams.getMarginTop();
+        }
+    }
+
+    @Override
+    public void onDraw(Component component, Canvas canvas) {
+        // onDraw method
```
Comment (samuel-abhishek) [inline pulldownlayout/src/main/java/com/w4lle/library/PullDownView.java]: If there is nothing to be implemented in the onDraw(), then DrawTask interface can be removed along with the overriden onDraw() method.

**Review by samuel-abhishek — CHANGES_REQUESTED** [review]
Please check the comments

File: .idea/.gitignore (hunk 1 above)
Comment (Jobin1000) [reply .idea/.gitignore]: Done

File: .gitignore (hunk 2 above)
Comment (Jobin1000) [reply .gitignore]: Done

File: pulldownlayout/src/main/java/com/w4lle/library/PullDownView.java (hunk 3 above)
Comment (Jobin1000) [reply pulldownlayout/src/main/java/com/w4lle/library/PullDownView.java]: Done

**Review by samuel-abhishek — APPROVED** [review]
Okay

**Review by sharathks84 — APPROVED** [review]
Okay Approved

**Review by Gleelakrishna — APPROVED** [review]
OK

**Pull request merged**